carapace-server/src/main/java/org/carapaceproxy/api/BackendsResource.java

```diff
@@ -130,10 +140,16 @@ public String getHttpBody() {
         Map<String, BackendHealthStatus> backendsSnapshot = backendHealthManager.getBackendsSnapshot();
 
         Map<String, BackendBean> res = new HashMap<>();
+        Collection<BackendConfiguration> backendsConf = server.getMapper().getBackends().values();
         for (Map.Entry<String, BackendHealthStatus> bb : backendsSnapshot.entrySet()) {
             EndpointKey key = EndpointKey.make(bb.getKey());
             EndpointStats epstats = stats.getEndpointStats(key);
-            BackendBean bean = new BackendBean(key.getHost(), key.getPort());
+            String id = backendsConf.stream()
```

Comment:
This is not so clear.
Backend id is the pk.
There is something wrong in this loop